/var/log/nova/nova-compute.log:2016-09-01 23:27:45.800 5337 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Attempting claim: memory 512 MB, disk 1 GB, vcpus 1 CPU
/var/log/nova/nova-compute.log:2016-09-01 23:27:45.801 5337 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Total memory: 48288 MB, used: 512.00 MB
/var/log/nova/nova-compute.log:2016-09-01 23:27:45.801 5337 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] memory limit: 144864.00 MB, free: 144352.00 MB
/var/log/nova/nova-compute.log:2016-09-01 23:27:45.801 5337 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Total disk: 51 GB, used: 0.00 GB
/var/log/nova/nova-compute.log:2016-09-01 23:27:45.801 5337 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] disk limit not specified, defaulting to unlimited
/var/log/nova/nova-compute.log:2016-09-01 23:27:45.802 5337 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Total vcpu: 24 VCPU, used: 0.00 VCPU
/var/log/nova/nova-compute.log:2016-09-01 23:27:45.802 5337 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] vcpu limit: 384.00 VCPU, free: 384.00 VCPU
/var/log/nova/nova-compute.log:2016-09-01 23:27:45.825 5337 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Claim successful
/var/log/nova/nova-compute.log:2016-09-01 23:27:46.307 5337 INFO nova.virt.libvirt.driver [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Creating image
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Instance failed to spawn
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Traceback (most recent call last):
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 2218, in _build_resources
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     yield resources
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 2064, in _build_and_run_instance
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     block_device_info=block_device_info)
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py", line 2773, in spawn
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     admin_pass=admin_password)
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py", line 3299, in _create_image
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     instance, network_info, admin_pass, files, suffix)
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py", line 3078, in _inject_data
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     network_info, libvirt_virt_type=CONF.libvirt.virt_type)
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/virt/netutils.py", line 78, in get_injected_network_template
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     if not (network_info and template):
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/model.py", line 517, in __len__
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     return self._sync_wrapper(fn, *args, **kwargs)
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/model.py", line 504, in _sync_wrapper
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     self.wait()
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/model.py", line 536, in wait
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     self[:] = self._gt.wait()
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/eventlet/greenthread.py", line 175, in wait
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     return self._exit_event.wait()
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/eventlet/event.py", line 121, in wait
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     return hubs.get_hub().switch()
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/eventlet/hubs/hub.py", line 294, in switch
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     return self.greenlet.switch()
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/eventlet/greenthread.py", line 214, in main
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     result = function(*args, **kwargs)
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/utils.py", line 1145, in context_wrapper
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     return func(*args, **kwargs)
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 1587, in _allocate_network_async
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     six.reraise(*exc_info)
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 1570, in _allocate_network_async
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     bind_host_id=bind_host_id)
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py", line 668, in allocate_for_instance
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     self._delete_ports(neutron, instance, created_port_ids)
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/oslo_utils/excutils.py", line 220, in __exit__
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     self.force_reraise()
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/oslo_utils/excutils.py", line 196, in force_reraise
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     six.reraise(self.type_, self.value, self.tb)
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py", line 657, in allocate_for_instance
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     security_group_ids, available_macs, dhcp_opts)
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py", line 259, in _create_port
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     raise exception.PortBindingFailed(port_id=port_id)
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] PortBindingFailed: Binding failed for port 03993376-46e0-497c-a18e-d228cca5fe3b, please check neutron logs for more information.
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.033 5337 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] 
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.034 5337 INFO nova.compute.manager [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Terminating instance
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.039 5337 INFO nova.virt.libvirt.driver [-] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] During wait destroy, instance disappeared.
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.054 5337 INFO nova.virt.libvirt.driver [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Deleting instance files /var/lib/nova/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1_del
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.055 5337 INFO nova.virt.libvirt.driver [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Deletion of /var/lib/nova/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1_del complete
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.155 5337 INFO nova.compute.manager [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Took 0.12 seconds to destroy the instance on the hypervisor.
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.211 5337 INFO nova.compute.manager [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Took 0.06 seconds to deallocate network for instance.
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.842 5332 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Attempting claim: memory 512 MB, disk 1 GB, vcpus 1 CPU
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.842 5332 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Total memory: 48288 MB, used: 512.00 MB
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.842 5332 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] memory limit: 144864.00 MB, free: 144352.00 MB
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.842 5332 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Total disk: 51 GB, used: 0.00 GB
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.843 5332 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] disk limit not specified, defaulting to unlimited
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.843 5332 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Total vcpu: 24 VCPU, used: 0.00 VCPU
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.843 5332 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] vcpu limit: 384.00 VCPU, free: 384.00 VCPU
/var/log/nova/nova-compute.log:2016-09-01 23:27:47.866 5332 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Claim successful
/var/log/nova/nova-compute.log:2016-09-01 23:27:48.307 5332 INFO nova.virt.libvirt.driver [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Creating image
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Instance failed to spawn
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Traceback (most recent call last):
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 2218, in _build_resources
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     yield resources
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 2064, in _build_and_run_instance
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     block_device_info=block_device_info)
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py", line 2773, in spawn
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     admin_pass=admin_password)
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py", line 3299, in _create_image
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     instance, network_info, admin_pass, files, suffix)
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py", line 3078, in _inject_data
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     network_info, libvirt_virt_type=CONF.libvirt.virt_type)
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/virt/netutils.py", line 78, in get_injected_network_template
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     if not (network_info and template):
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/model.py", line 517, in __len__
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     return self._sync_wrapper(fn, *args, **kwargs)
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/model.py", line 504, in _sync_wrapper
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     self.wait()
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/model.py", line 536, in wait
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     self[:] = self._gt.wait()
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/eventlet/greenthread.py", line 175, in wait
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     return self._exit_event.wait()
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/eventlet/event.py", line 121, in wait
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     return hubs.get_hub().switch()
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/eventlet/hubs/hub.py", line 294, in switch
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     return self.greenlet.switch()
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/eventlet/greenthread.py", line 214, in main
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     result = function(*args, **kwargs)
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/utils.py", line 1145, in context_wrapper
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     return func(*args, **kwargs)
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 1587, in _allocate_network_async
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     six.reraise(*exc_info)
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 1570, in _allocate_network_async
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     bind_host_id=bind_host_id)
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py", line 668, in allocate_for_instance
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     self._delete_ports(neutron, instance, created_port_ids)
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/oslo_utils/excutils.py", line 220, in __exit__
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     self.force_reraise()
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/oslo_utils/excutils.py", line 196, in force_reraise
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     six.reraise(self.type_, self.value, self.tb)
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py", line 657, in allocate_for_instance
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     security_group_ids, available_macs, dhcp_opts)
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py", line 259, in _create_port
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     raise exception.PortBindingFailed(port_id=port_id)
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] PortBindingFailed: Binding failed for port d157bab8-0129-4a9e-a1e5-f3f8920d88e5, please check neutron logs for more information.
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.364 5332 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] 
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.365 5332 INFO nova.compute.manager [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Terminating instance
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.371 5332 INFO nova.virt.libvirt.driver [-] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] During wait destroy, instance disappeared.
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.384 5332 INFO nova.virt.libvirt.driver [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Deleting instance files /var/lib/nova/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1_del
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.385 5332 INFO nova.virt.libvirt.driver [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Deletion of /var/lib/nova/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1_del complete
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.600 5332 INFO nova.compute.manager [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Took 0.23 seconds to destroy the instance on the hypervisor.
/var/log/nova/nova-compute.log:2016-09-01 23:27:49.653 5332 INFO nova.compute.manager [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Took 0.05 seconds to deallocate network for instance.
/var/log/nova/nova-compute.log:2016-09-01 23:27:50.281 5356 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Attempting claim: memory 512 MB, disk 1 GB, vcpus 1 CPU
/var/log/nova/nova-compute.log:2016-09-01 23:27:50.281 5356 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Total memory: 48288 MB, used: 512.00 MB
/var/log/nova/nova-compute.log:2016-09-01 23:27:50.282 5356 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] memory limit: 144864.00 MB, free: 144352.00 MB
/var/log/nova/nova-compute.log:2016-09-01 23:27:50.282 5356 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Total disk: 51 GB, used: 0.00 GB
/var/log/nova/nova-compute.log:2016-09-01 23:27:50.282 5356 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] disk limit not specified, defaulting to unlimited
/var/log/nova/nova-compute.log:2016-09-01 23:27:50.282 5356 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Total vcpu: 24 VCPU, used: 0.00 VCPU
/var/log/nova/nova-compute.log:2016-09-01 23:27:50.283 5356 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] vcpu limit: 384.00 VCPU, free: 384.00 VCPU
/var/log/nova/nova-compute.log:2016-09-01 23:27:50.308 5356 INFO nova.compute.claims [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Claim successful
/var/log/nova/nova-compute.log:2016-09-01 23:27:50.723 5356 INFO nova.virt.libvirt.driver [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Creating image
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Instance failed to spawn
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Traceback (most recent call last):
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 2218, in _build_resources
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     yield resources
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 2064, in _build_and_run_instance
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     block_device_info=block_device_info)
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py", line 2773, in spawn
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     admin_pass=admin_password)
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py", line 3299, in _create_image
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     instance, network_info, admin_pass, files, suffix)
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/virt/libvirt/driver.py", line 3078, in _inject_data
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     network_info, libvirt_virt_type=CONF.libvirt.virt_type)
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/virt/netutils.py", line 78, in get_injected_network_template
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     if not (network_info and template):
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/model.py", line 517, in __len__
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     return self._sync_wrapper(fn, *args, **kwargs)
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/model.py", line 504, in _sync_wrapper
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     self.wait()
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/model.py", line 536, in wait
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     self[:] = self._gt.wait()
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/eventlet/greenthread.py", line 175, in wait
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     return self._exit_event.wait()
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/eventlet/event.py", line 121, in wait
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     return hubs.get_hub().switch()
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/eventlet/hubs/hub.py", line 294, in switch
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     return self.greenlet.switch()
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/eventlet/greenthread.py", line 214, in main
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     result = function(*args, **kwargs)
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/utils.py", line 1145, in context_wrapper
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     return func(*args, **kwargs)
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 1587, in _allocate_network_async
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     six.reraise(*exc_info)
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 1570, in _allocate_network_async
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     bind_host_id=bind_host_id)
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py", line 668, in allocate_for_instance
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     self._delete_ports(neutron, instance, created_port_ids)
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/oslo_utils/excutils.py", line 220, in __exit__
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     self.force_reraise()
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/oslo_utils/excutils.py", line 196, in force_reraise
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     six.reraise(self.type_, self.value, self.tb)
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py", line 657, in allocate_for_instance
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     security_group_ids, available_macs, dhcp_opts)
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]   File "/usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py", line 259, in _create_port
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1]     raise exception.PortBindingFailed(port_id=port_id)
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] PortBindingFailed: Binding failed for port 6af8ee28-468f-461d-86b6-b037e2abd8c2, please check neutron logs for more information.
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.648 5356 ERROR nova.compute.manager [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] 
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.650 5356 INFO nova.compute.manager [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Terminating instance
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.653 5356 INFO nova.virt.libvirt.driver [-] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] During wait destroy, instance disappeared.
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.673 5356 INFO nova.virt.libvirt.driver [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Deleting instance files /var/lib/nova/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1_del
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.674 5356 INFO nova.virt.libvirt.driver [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Deletion of /var/lib/nova/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1_del complete
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.773 5356 INFO nova.compute.manager [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Took 0.12 seconds to destroy the instance on the hypervisor.
/var/log/nova/nova-compute.log:2016-09-01 23:27:51.833 5356 INFO nova.compute.manager [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Took 0.06 seconds to deallocate network for instance.
/var/log/httpd/horizon_ssl_access.log:10.212.134.200 - - [01/Sep/2016:23:27:48 -0400] "GET /dashboard/project/instances/?action=row_update&table=instances&obj_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 1311 "https://10.1.13.251/dashboard/project/instances/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.89 Safari/537.36"
/var/log/httpd/horizon_ssl_access.log:10.212.134.200 - - [01/Sep/2016:23:27:51 -0400] "GET /dashboard/project/instances/?action=row_update&table=instances&obj_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 1310 "https://10.1.13.251/dashboard/project/instances/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.89 Safari/537.36"
/var/log/httpd/horizon_ssl_access.log:10.212.134.200 - - [01/Sep/2016:23:27:52 -0400] "GET /dashboard/project/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/ HTTP/1.1" 200 5431 "https://10.1.13.251/dashboard/project/instances/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.89 Safari/537.36"
/var/log/httpd/horizon_ssl_access.log:10.212.134.200 - - [01/Sep/2016:23:27:53 -0400] "GET /dashboard/static/dashboard/css/34f8a8f8d5e5.css HTTP/1.1" 200 4428 "https://10.1.13.251/dashboard/project/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.89 Safari/537.36"
/var/log/httpd/horizon_ssl_access.log:10.212.134.200 - - [01/Sep/2016:23:27:53 -0400] "GET /dashboard/static/dashboard/css/f3e16c56e573.css HTTP/1.1" 200 53336 "https://10.1.13.251/dashboard/project/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.89 Safari/537.36"
/var/log/httpd/horizon_ssl_access.log:10.212.134.200 - - [01/Sep/2016:23:27:53 -0400] "GET /dashboard/static/dashboard/js/7fd3f7d69c71.js HTTP/1.1" 200 133341 "https://10.1.13.251/dashboard/project/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.89 Safari/537.36"
/var/log/httpd/horizon_ssl_access.log:10.212.134.200 - - [01/Sep/2016:23:27:53 -0400] "GET /dashboard/i18n/js/horizon+openstack_dashboard/ HTTP/1.1" 200 2372 "https://10.1.13.251/dashboard/project/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.89 Safari/537.36"
/var/log/httpd/horizon_ssl_access.log:10.212.134.200 - - [01/Sep/2016:23:27:53 -0400] "GET /dashboard/static/dashboard/js/5ac31e075a3b.js HTTP/1.1" 200 289134 "https://10.1.13.251/dashboard/project/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.89 Safari/537.36"
/var/log/httpd/horizon_ssl_access.log:10.212.134.200 - - [01/Sep/2016:23:27:54 -0400] "GET /dashboard/static/framework/widgets/toast/toast.html HTTP/1.1" 200 180 "https://10.1.13.251/dashboard/project/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.89 Safari/537.36"
/var/log/httpd/horizon_ssl_access.log:10.212.134.200 - - [01/Sep/2016:23:32:39 -0400] "GET /dashboard/project/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/?tab=instance_details__log HTTP/1.1" 200 385 "https://10.1.13.251/dashboard/project/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.89 Safari/537.36"
/var/log/httpd/horizon_ssl_access.log:10.212.134.200 - - [01/Sep/2016:23:32:42 -0400] "GET /dashboard/project/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/?tab=instance_details__audit HTTP/1.1" 200 532 "https://10.1.13.251/dashboard/project/instances/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.89 Safari/537.36"
/var/log/neutron/server.log:2016-09-01 23:27:45.702 16696 INFO neutron.wsgi [req-ee9531fd-699a-4669-b798-db9b6d8235d5 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:27:45] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.067614
/var/log/neutron/server.log:2016-09-01 23:27:46.141 16681 INFO neutron.wsgi [req-a2609738-5ccd-4e56-bc81-60129cdf2fa6 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:27:46] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.060728
/var/log/neutron/server.log:2016-09-01 23:27:46.195 16700 INFO neutron.wsgi [req-5dc6ec1d-552b-4418-8bb0-070b9910260b a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:27:46] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.047516
/var/log/neutron/server.log:2016-09-01 23:27:47.196 16678 INFO neutron.wsgi [req-25b001b9-1710-4977-9125-3215cbc85e25 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.13 - - [01/Sep/2016 23:27:47] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.038640
/var/log/neutron/server.log:2016-09-01 23:27:48.406 16681 INFO neutron.wsgi [req-f04808ed-df6a-4d2b-8f7d-c0849b8de093 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:27:48] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.027328
/var/log/neutron/server.log:2016-09-01 23:27:48.517 16693 INFO neutron.wsgi [req-c4e746ff-17fd-4a71-9236-154b06c22212 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:27:48] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.028198
/var/log/neutron/server.log:2016-09-01 23:27:49.308 16678 INFO neutron.notifiers.nova [-] Nova event response: {u'status': u'completed', u'tag': u'03993376-46e0-497c-a18e-d228cca5fe3b', u'name': u'network-vif-deleted', u'server_uuid': u'415b6af1-e47a-4ac6-912f-aeddd9d5d1b1', u'code': 200}
/var/log/neutron/server.log:2016-09-01 23:27:49.639 16693 INFO neutron.wsgi [req-cdc27dbe-1f6f-4f5f-9689-c5822d1b6b2b a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.12 - - [01/Sep/2016 23:27:49] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.036858
/var/log/neutron/server.log:2016-09-01 23:27:51.426 16693 INFO neutron.notifiers.nova [-] Nova event response: {u'status': u'completed', u'tag': u'd157bab8-0129-4a9e-a1e5-f3f8920d88e5', u'name': u'network-vif-deleted', u'server_uuid': u'415b6af1-e47a-4ac6-912f-aeddd9d5d1b1', u'code': 200}
/var/log/neutron/server.log:2016-09-01 23:27:51.815 16696 INFO neutron.wsgi [req-4c64d8be-1a51-4a62-a8c6-b8a10f27430d a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.11 - - [01/Sep/2016 23:27:51] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.039162
/var/log/neutron/server.log:2016-09-01 23:27:52.001 16682 INFO neutron.wsgi [req-743a11c2-4cea-4928-b7f6-eebcaa8401c3 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:27:52] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.034306
/var/log/neutron/server.log:2016-09-01 23:27:52.057 16700 INFO neutron.wsgi [req-27b43726-fa21-4969-9e55-d8fd3cb7f410 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:27:52] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.027500
/var/log/neutron/server.log:2016-09-01 23:27:52.203 16699 INFO neutron.wsgi [req-7d6720d5-1c60-49f5-8fc4-a2869e3ffd30 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:27:52] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.059440
/var/log/neutron/server.log:2016-09-01 23:27:52.698 16691 INFO neutron.wsgi [req-3ff65dd1-538d-47d9-b64e-14daaaf61852 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:27:52] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.034304
/var/log/neutron/server.log:2016-09-01 23:27:52.918 16677 INFO neutron.wsgi [req-94428cfe-9f4f-4430-b54f-fa5ed01a347d a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:27:52] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.030995
/var/log/neutron/server.log:2016-09-01 23:27:52.947 16677 INFO neutron.wsgi [req-bfec1161-0e19-4296-883a-8d3438027d89 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:27:52] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.026476
/var/log/neutron/server.log:2016-09-01 23:30:56.342 16694 INFO neutron.wsgi [req-d3f69a00-c7f2-45f0-8870-17cfcff8ceea a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:30:56] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.248189
/var/log/neutron/server.log:2016-09-01 23:31:44.968 16682 INFO neutron.wsgi [req-15322b67-b324-402d-8be5-67fbc8b0f515 93e10f1269af498e9791f2baebe91f72 e5275a07edc54c9c97b5b304fe3b1b9a - - -] 10.1.13.251 - - [01/Sep/2016 23:31:44] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1&device_id=e19e5bb8-ebdd-48c2-aa60-83339a7a8913 HTTP/1.1" 200 226 0.159376
/var/log/neutron/server.log:2016-09-01 23:32:39.510 16693 INFO neutron.wsgi [req-334fa507-1fac-4dac-a141-7c38fcd0323e a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:32:39] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.062128
/var/log/neutron/server.log:2016-09-01 23:32:39.756 16691 INFO neutron.wsgi [req-860721f2-8f54-4219-95f6-eddd6b6cb247 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:32:39] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.037236
/var/log/neutron/server.log:2016-09-01 23:32:39.786 16691 INFO neutron.wsgi [req-b0dbe7f0-f572-4db8-86ee-791e11f16d68 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:32:39] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.026935
/var/log/neutron/server.log:2016-09-01 23:32:43.256 16696 INFO neutron.wsgi [req-efc9cb54-115b-4cb0-be74-cc747926ff4c a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:32:43] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.034297
/var/log/neutron/server.log:2016-09-01 23:32:43.468 16685 INFO neutron.wsgi [req-f97aa853-362f-4cf3-942c-99cb0b7bc99d a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:32:43] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.035932
/var/log/neutron/server.log:2016-09-01 23:32:43.497 16685 INFO neutron.wsgi [req-3794162c-f7cb-410a-88f6-e2da63599873 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:32:43] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.026499
/var/log/neutron/server.log:2016-09-01 23:35:56.531 16679 INFO neutron.wsgi [req-533334d6-7960-4e99-ae60-00a082ff2849 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 - - [01/Sep/2016 23:35:56] "GET /v2.0/ports.json?device_id=415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" 200 226 0.138444
/var/log/nova/nova-api.log:2016-09-01 23:27:45.705 15097 INFO nova.osapi_compute.wsgi.server [req-1069e3b1-6076-4709-a9d3-db8f2f5a9a0a a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 "GET /v2/4609a26e18504a5e828b2f02a95f3014/servers/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" status: 200 len: 1615 time: 0.2150998
/var/log/nova/nova-api.log:2016-09-01 23:27:48.409 15113 INFO nova.osapi_compute.wsgi.server [req-d6974bcc-5e04-41d1-b311-447268c18888 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 "GET /v2/4609a26e18504a5e828b2f02a95f3014/servers/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" status: 200 len: 1699 time: 0.3415799
/var/log/nova/nova-api.log:2016-09-01 23:27:49.256 15095 INFO nova.api.openstack.compute.server_external_events [req-dd04af9d-ae18-4c78-bbe3-31f565b2051e 9d5823fd27604d129562e27e02e9cb6a e5275a07edc54c9c97b5b304fe3b1b9a - - -] Creating event network-vif-deleted:03993376-46e0-497c-a18e-d228cca5fe3b for instance 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1
/var/log/nova/nova-api.log:2016-09-01 23:27:51.420 15095 INFO nova.api.openstack.compute.server_external_events [req-11024db4-4f7d-42ab-b68a-b146fc8cf0da 9d5823fd27604d129562e27e02e9cb6a e5275a07edc54c9c97b5b304fe3b1b9a - - -] Creating event network-vif-deleted:d157bab8-0129-4a9e-a1e5-f3f8920d88e5 for instance 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1
/var/log/nova/nova-api.log:2016-09-01 23:27:52.003 15101 INFO nova.osapi_compute.wsgi.server [req-419db2d4-df7d-4121-841a-5a1134735d69 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 "GET /v2/4609a26e18504a5e828b2f02a95f3014/servers/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" status: 200 len: 1613 time: 0.1382020
/var/log/nova/nova-api.log:2016-09-01 23:27:52.700 15108 INFO nova.osapi_compute.wsgi.server [req-85cc0e04-1f6f-45ef-91f9-42d2e6c44552 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 "GET /v2/4609a26e18504a5e828b2f02a95f3014/servers/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" status: 200 len: 2219 time: 0.1890740
/var/log/nova/nova-api.log:2016-09-01 23:27:52.819 15118 INFO nova.osapi_compute.wsgi.server [req-a6c83992-b3bf-41ed-994e-85376189681b a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 "GET /v2/4609a26e18504a5e828b2f02a95f3014/servers/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/os-volume_attachments HTTP/1.1" status: 200 len: 221 time: 0.1152570
/var/log/nova/nova-api.log:2016-09-01 23:32:39.513 15118 INFO nova.osapi_compute.wsgi.server [req-ea3bf1d3-919f-458d-9c2b-549690b19d89 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 "GET /v2/4609a26e18504a5e828b2f02a95f3014/servers/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" status: 200 len: 2219 time: 0.1829531
/var/log/nova/nova-api.log:2016-09-01 23:32:39.641 15093 INFO nova.osapi_compute.wsgi.server [req-cb5e9e3b-a557-4c85-b398-6e8660ad7950 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 "GET /v2/4609a26e18504a5e828b2f02a95f3014/servers/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/os-volume_attachments HTTP/1.1" status: 200 len: 221 time: 0.1220422
/var/log/nova/nova-api.log:2016-09-01 23:32:40.087 15103 INFO nova.api.openstack.wsgi [req-369e03fe-d668-4ee0-86f1-19aff2835b9f a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] HTTP exception thrown: Instance 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 is not ready
/var/log/nova/nova-api.log:2016-09-01 23:32:40.089 15103 INFO nova.osapi_compute.wsgi.server [req-369e03fe-d668-4ee0-86f1-19aff2835b9f a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 "POST /v2/4609a26e18504a5e828b2f02a95f3014/servers/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/action HTTP/1.1" status: 409 len: 328 time: 0.1301470
/var/log/nova/nova-api.log:2016-09-01 23:32:43.257 15097 INFO nova.osapi_compute.wsgi.server [req-0ba5afe4-25bf-485e-a0ad-78206f4ce343 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 "GET /v2/4609a26e18504a5e828b2f02a95f3014/servers/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 HTTP/1.1" status: 200 len: 2219 time: 0.3014250
/var/log/nova/nova-api.log:2016-09-01 23:32:43.335 15097 INFO nova.osapi_compute.wsgi.server [req-64fb0406-02b7-49a3-9db6-0e319cbebee8 a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 "GET /v2/4609a26e18504a5e828b2f02a95f3014/servers/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/os-volume_attachments HTTP/1.1" status: 200 len: 221 time: 0.0734770
/var/log/nova/nova-api.log:2016-09-01 23:32:43.806 15108 INFO nova.osapi_compute.wsgi.server [req-780fb0f3-281a-4ffd-8bf5-385cbcdb8e2c a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] 10.1.13.251 "GET /v2/4609a26e18504a5e828b2f02a95f3014/servers/415b6af1-e47a-4ac6-912f-aeddd9d5d1b1/os-instance-actions HTTP/1.1" status: 200 len: 513 time: 0.0896089
/var/log/nova/nova-conductor.log:2016-09-01 23:27:47.436 15913 ERROR nova.scheduler.utils [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Error from last host: stack3.orbit8.com (node stack3.orbit8.com): [u'Traceback (most recent call last):\n', u'  File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 1926, in _do_build_and_run_instance\n    filter_properties)\n', u'  File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 2116, in _build_and_run_instance\n    instance_uuid=instance.uuid, reason=six.text_type(e))\n', u'RescheduledException: Build of instance 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 was re-scheduled: Binding failed for port 03993376-46e0-497c-a18e-d228cca5fe3b, please check neutron logs for more information.\n']
/var/log/nova/nova-conductor.log:2016-09-01 23:27:49.971 15903 ERROR nova.scheduler.utils [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Error from last host: stack2.orbit8.com (node stack2.orbit8.com): [u'Traceback (most recent call last):\n', u'  File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 1926, in _do_build_and_run_instance\n    filter_properties)\n', u'  File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 2116, in _build_and_run_instance\n    instance_uuid=instance.uuid, reason=six.text_type(e))\n', u'RescheduledException: Build of instance 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 was re-scheduled: Binding failed for port d157bab8-0129-4a9e-a1e5-f3f8920d88e5, please check neutron logs for more information.\n']
/var/log/nova/nova-conductor.log:2016-09-01 23:27:52.067 15902 ERROR nova.scheduler.utils [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Error from last host: stack1.orbit8.com (node stack1.orbit8.com): [u'Traceback (most recent call last):\n', u'  File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 1926, in _do_build_and_run_instance\n    filter_properties)\n', u'  File "/usr/lib/python2.7/site-packages/nova/compute/manager.py", line 2116, in _build_and_run_instance\n    instance_uuid=instance.uuid, reason=six.text_type(e))\n', u'RescheduledException: Build of instance 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1 was re-scheduled: Binding failed for port 6af8ee28-468f-461d-86b6-b037e2abd8c2, please check neutron logs for more information.\n']
/var/log/nova/nova-conductor.log:2016-09-01 23:27:52.068 15902 WARNING nova.scheduler.utils [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] Failed to compute_task_build_instances: Exceeded maximum number of retries. Exceeded max scheduling attempts 3 for instance 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1. Last exception: Binding failed for port 6af8ee28-468f-461d-86b6-b037e2abd8c2, please check neutron logs for more information.
/var/log/nova/nova-conductor.log:2016-09-01 23:27:52.068 15902 WARNING nova.scheduler.utils [req-efd02b29-5507-4aff-b9fc-b069261cfcfd a01a11068ae742de856b960cad5860f2 4609a26e18504a5e828b2f02a95f3014 - - -] [instance: 415b6af1-e47a-4ac6-912f-aeddd9d5d1b1] Setting instance to ERROR state.
